### PR TITLE
Oracle_30974: Ingest pipeline date parsing issue

### PR DESCRIPTION
**Pull request by ishleenk17**
## What does this PR do?

This PR parses a 2 digit month day format for the oracle's ingest pipeline

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

The test log file can be changed to a 2 month day. Without the change the particular changed log will not be parsed. With the change, the parsing of the 2 month day will happen successfully.

## Related issues

- Closes https://github.com/elastic/beats/issues/30974

### CODE REVIEW

File: packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Wed Oct  7 11:58:08 2020 -04:00
+Wed Oct  21 11:58:08 2020 -04:00
```
Comment (jsoriano) [inline packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log:1]: Spacing in the Beats PR is different, can you confirm if Oracle logs with one or two spaces before a double-digit day?

In Beats: https://github.com/elastic/beats/pull/31514/files#diff-6817ecbfad5bf76c9dfddcb39ac861194449af8b4ee4af1060ad67077280cd80R16

```suggestion
Wed Oct 21 11:58:08 2020 -04:00
```

**Comment by ishleenk17** [comment]
Yes @jsoriano , it's a single spaces before double digit date. Made the needed change

File: packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log-expected.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
             "ecs": {
                 "version": "8.0.0"
             },
             "event": {
                 "action": "database_audit",
                 "category": "database",
                 "kind": "event",
-                "original": "Wed Oct  7 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
+                "original": "Wed Oct  21 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
```
Comment (jsoriano) [inline packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log-expected.json:12]: This file should be modified or regenerated too.
```suggestion
                "original": "Wed Oct 21 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
```

**Review by andrewkroh — APPROVED** [review]

**Comment by andrewkroh** [comment]
How does this differ from https://github.com/elastic/integrations/pull/3337?

**Pull request merged**